src/dmd/identifier.h

```diff
@@ -35,7 +35,7 @@ class Identifier : public RootObject
 
     static inline Identifier *idPool(const char *s)
     {
-        return idPool(s, strlen(s));
+        return idPool(s, static_cast<unsigned>(strlen(s)));
```

Comment:
Which port is broken here?

Reply:
This is just to prevent warnings (size_t -> unsigned truncations). IIRC, someone adjusted the `idPool()` signature due to OSX `size_t` mangling issues with some DMD versions.